net-vpn/logmein-hamachi/logmein-hamachi-2.1.0.198.ebuild

```diff
@@ -9,9 +9,9 @@ HOMEPAGE="https://www.vpn.net/"
 SRC_URI="x86?	( https://www.vpn.net/installers/${P}-x86.tgz )
 	amd64?	( https://www.vpn.net/installers/${P}-x64.tgz )"
 
-LICENSE="LogMeIn"
+LICENSE="LogMeIn-2017"
 SLOT="0"
-KEYWORDS="-* ~amd64 ~x86"
+KEYWORDS="-* amd64 ~x86"
```

Comment:
You can not perform this action. Only members of arch team can do the stabilization. Just focus on renaming the license, and request a stabilization for this version through bugzilla.

Reply:
I reverted the stabilization part, thanks